Assets/_BForBoss/_Weapons/Scripts/Weapons/WeaponsManager.cs

```diff
@@ -63,9 +68,15 @@ public class WeaponsManager : MonoBehaviour
             _isDashing = isDashing;
             _isWallRunning = isWallRunning;
         }
+        
+        public void MeleeHitCallback()
+        {
+            _equipmentBehaviour.ApplyMeleeDamageDelayed();
+        }
 
         private void Start()
         {
+            _weaponAnimator = GetComponentInChildren<Animator>();
```

Comment:
Up to you, but does this have to be on `Start()` or can we just attach an Animator to `_weaponAnimator` via `[SerializeField]`